app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,5 @@
+<div id="post" class="border-solid border-4 border-black md:w-1/2 mx-auto w-full mb-5">
+  <h2 class="font-bold ml-3"><%= post.title %></h2>
+  <p class="mt-5 ml-3" ><%= post.text %></p>
+  <p class="mt-5 right-2 ml-3">comments: <span class="font-bold"><%= post.comments_counter%>, </span>Likes: <span class="font-bold"><%= post.likes_counter %></span></p>
```

Comment:
- [ ] Please give some space around the components as in the given wireframe, also the counters should be on the right bottom.
![image](https://github.com/rmiyoyo/blog-app/assets/91486791/22d0e0d2-da00-417f-b50c-f9a0b0fae112)
![image](https://github.com/rmiyoyo/blog-app/assets/91486791/1782cf33-bbcf-492f-8361-a28b5dcd0aa9)